src/container/srv_container.c

```diff
@@ -2025,6 +2164,7 @@ cont_open(struct rdb_tx *tx, struct ds_pool_hdl *pool_hdl, struct cont *cont,
 	chdl.ch_flags = in->coi_flags;
 	chdl.ch_sec_capas = sec_capas;
 
+	d_iov_set(&value, &chdl, sizeof(chdl));
```

Comment:
is this identical d_iov_set() call for &value for code readability vs. correctness? It's a good improvement I think if it is for readability only, just asking the question.